rules/integrations/o365/collection_microsoft_365_new_inbox_rule.toml

```diff
@@ -43,9 +43,9 @@ query = '''
 event.dataset:o365.audit and event.provider:Exchange and
 event.category:web and event.action:"New-InboxRule" and
     (
-        o365audit.Parameters.ForwardTo:* or
-        o365audit.Parameters.ForwardAsAttachmentTo:* or
-        o365audit.Parameters.RedirectTo:*
+        o365.audit.Parameters.ForwardTo:* or
```

Comment:
so did these change or were they always wrong?

Reply:
I've introduced them when I tuned the logic of this rule.
PR: https://github.com/elastic/detection-rules/pull/1683

Reply:
oh, so the correct fields may exist in the parsed filebeat schema. If you remove those fields from the non-ecs schema, does this pass

Reply:
I'll try it